tests/sls/homu/valid_repos.py

```diff
@@ -0,0 +1,39 @@
+from urllib.error import URLError
+from urllib.request import Request, urlopen
+
+import toml
+
+from tests.util import Failure, Success
+
+
+def repoExists(identifier):
+    '''
+    Consumes a repo identifier string in the form of owner/name and returns a
```

Comment:
No need to repeat all the details of how the checking is done, the docstring only needs to describe the overall goal of the method, e.g. `Checks if the given repo exists on GitHub`.